src/mongoose_redis.erl

```diff
@@ -0,0 +1,84 @@
+%%%-------------------------------------------------------------------
+%%% @doc
+%%%
+%%% @end
+%%% Created : 27. Jun 2018 17:16
+%%%-------------------------------------------------------------------
+-module(mongoose_redis).
+-author("bartlomiej.gorny@erlang-solutions.com").
+-include("mongoose.hrl").
+-define(POOL_NAME, redis_pool).
+
+%% API
+-export([start_pool/1, cmd/1, cmd/2, cmds/1, cmds/2]).
+
+%%%===================================================================
+%%% API
+%%%===================================================================
+
+-spec start_pool(list()) -> {ok, pid()} | {error, {already_started, pid()}}.
+start_pool(Opts) ->
+    mongoose_wpool:setup_env(),
+    PoolSize = proplists:get_value(pool_size, Opts, 10),
+    RedisOpts = proplists:get_value(worker_config, Opts, []),
+    PoolOpts = proplists:get_value(pool_opts, Opts, []),
+    PoolOptions = [{workers, PoolSize},
+                   {worker, {eredis_client, makeargs(RedisOpts)}}
+                  ] ++ PoolOpts,
+    Res = wpool:start_sup_pool(?POOL_NAME, PoolOptions),
+    case Res of
+        {ok, _Pid} -> Res;
+        Error ->
+            ?ERROR_MSG("Failed to start worker pool, reason: ~p~n", [Error]),
+            Error
+    end.
+
+-spec cmd(iolist()) -> undefined
```

Comment:
This and following specs are not very readable. Could you use syntax like the following:
```erlang
-spec cmd(iolist()) -> Result when
    Result :: binary() | ....
```

It may improve the readability here.

It'd be even better if you could re-use types from `eredis.hrl`.